app/src/main/java/com/steamclock/steamclogsample/MainActivity.kt

```diff
@@ -48,45 +55,98 @@ class MainActivity : AppCompatActivity() {
                 id: Long
             ) {
                 clog.config.logLevel = when (position) {
-                    0 -> LogLevelPreset.Firehose
-                    1 -> LogLevelPreset.Develop
-                    2 -> LogLevelPreset.Release
-                    3 -> LogLevelPreset.ReleaseAdvanced
-                    else -> LogLevelPreset.Firehose
+                    0 -> {
+                        LogLevelPreset.Firehose
+                    }
+                    1 -> {
+                        LogLevelPreset.Develop
+                    }
+                    2 -> {
+                        LogLevelPreset.Release
+                    }
+                    3 -> {
+                        LogLevelPreset.ReleaseAdvanced
+                    }
+                    else -> {
+                        LogLevelPreset.Firehose
+                    }
                 }
+                clog.warn("LogLevel changed to ${clog.config.logLevel.title}")
             }
         }
     }
 
-    private fun testAllLoggingLevels() {
+    private fun testLogging() {
+        testVerbose()
+        testDebug()
+        testWarn()
+        testInfo()
+        Toast.makeText(applicationContext, "Logged some things! Check your console or show the dump the log.", Toast.LENGTH_LONG).show()
+    }
+
+    private fun testVerbose() {
         clog.verbose("Verbose message")
-        clog.verbose("Verbose message", RedactableParent())
+        clog.verbose("Verbose message with data", RedactableParent())
+    }
 
+    private fun testDebug() {
         clog.debug("Debug message")
-        clog.debug("Debug message", RedactableParent())
+        clog.debug("Debug message with data", RedactableParent())
+    }
 
+    private fun testInfo() {
         clog.info("Info message")
-        clog.info("Info message", RedactableParent())
+        clog.info("Info message with data", RedactableParent())
+    }
 
+    private fun testWarn() {
         clog.warn("Warn message")
-        clog.warn("Warn message", RedactableParent())
-
-        Toast.makeText(applicationContext, "Logged some things! Check your console or show the dump the log.", Toast.LENGTH_LONG).show()
+        clog.warn("Warn message with data", RedactableParent())
     }
 
     private fun testNonFatal() {
-        clog.error("Error message")
-        clog.error("Error message", RedactableParent())
-        clog.error("Error message", Throwable("OriginalNonFatalThrowable"))
-        clog.error("Error message", Throwable("OriginalNonFatalThrowable"), RedactableParent())
+        // Won't create new tickets
+        clog.info("Running testNonFatal")
+
+        // Should create tickets
+        //
+        // NOTE, Sentry seems to apply some grouping logic such that the *same* Throwable thrown
```

Comment:
We might need a way to override this behaviour later? I know iOS has the ability to send in a non-fatal as a "user report", which always needs to generate its own issue.

Reply:
Ok, I'll create a ticket for that. I think that given the changes I made here it should *hopefully* be easy to support as long as the "Message" used to create the user report is unique. We would use `Sentry.captureMessage()` instead of `Sentry. captureException()` which I think would avoid this "grouping" case.

Reply:
https://github.com/steamclock/steamclog-android/issues/56